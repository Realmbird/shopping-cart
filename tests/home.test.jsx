import { describe, it, expect } from 'vitest';
import { render, screen } from "@testing-library/react";
import Home from '../src/components/home.jsx'

describe("Home", () => {
    it("renders", () => {
        const { container } = render(<Home />);
        expect(container).toMatchSnapshot();
    })
})