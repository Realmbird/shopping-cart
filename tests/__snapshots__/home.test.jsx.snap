// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Home > renders 1`] = `
<div>
  <div>
    <div
      class="_home_693d29"
    >
      <div
        class="_title_693d29"
      >
        <img
          alt="logo"
          class="_logo_693d29"
          src="/src/assets/logo-site.png"
        />
        <h1>
          Awesome Shop
        </h1>
      </div>
    </div>
  </div>
</div>
`;
